assets/js/apps/project_detail_view/components/ExperimentList/ExperimentList.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from "react";
+import "./ExperimentList.css";
+import { Alert, Spinner } from "react-bootstrap";
+
+export default function ExperimentList(props) {
+  let [experimentsData, setExperimentsData] = useState(null);
+  let [resultLimit, setResultLimit] = useState(2000);
+
+  // fetching experiment data by project id
+  useEffect(() => {
+    fetch(`/api/v1/experiment/?limit=${resultLimit}&project__id=${props.projectId}`, {
+      method: 'get',
+      headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json'
+      }
+    }).then(response => {
+      console.log('experiment by project fetch response.')
+      if (!response.ok) {
+        throw new Error("An error occurred fetching experiments.")
+      }
+      console.log("experiment fetch response was OK.")
+      response.json().then(experimentsData => {
+        setExperimentsData(experimentsData)
+      })
+    })
+  }, [])
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933948)
```suggestion
    }, [])
```